src/app/AppNavItem.vue

```diff
@@ -117,21 +122,13 @@ const isActive = computed(() => {
     return true
   }
 
-  const currentRouteSubpath = route.path.split('/')[2]
-  if (currentRouteSubpath === targetRoute.value.name) {
+  const hasAnchorRoute = props.anchorRouteName !== '' && route.matched.some((matchedRoute) => matchedRoute.name === props.anchorRouteName)
```

Comment:
**Comment**: Save from listing _all_ routes in `src/app/getNavItems.ts` that belong to a nav item, this is the simplest I could come up with.

It turns out we will already need to add names to routes that we don’t necessarily want to name because of https://github.com/vuejs/router/issues/1829. TL;DR: unnamed and unresolved routes (i.e. what `route.matched` will contain) can’t be resolved using `router.resolve` so naming them is necessary to get breadcrumb generation to work.

In that sense the only additional work is manually stating the anchor route when defining nav items.